composer.json

```diff
@@ -11,14 +11,14 @@
         "php": "^7.3|^8.0",
```

Comment:
If this PR mostly addresses PHP 8 compatibility couldn't we adhere to PHP 8 only as dependency?